doc/nrf/ug_zigbee_configuring.rst

```diff
@@ -149,7 +149,27 @@ To customize them, use the following options:
 * :option:`CONFIG_ZBOSS_TRACE_MASK` - Sets the modules from which ZBOSS will log the debug messages with :option:`CONFIG_ZBOSS_TRACE_LOG_LEVEL`; no module is set by default.
 * :option:`CONFIG_ZBOSS_TRAF_DUMP` - Enables logging of the received 802.15.4 frames over ZBOSS trace log if :option:`CONFIG_ZBOSS_TRACE_LOG_LEVEL` is set; disabled by default.
 
-The stack logs are provided in a binary format (hex dump).
+The stack logs are provided in a binary format and you can configure how they are printed:
+
+* :option:`CONFIG_ZBOSS_TRACE_HEXDUMP_LOGGING` - Stack logs are printed as hexdump using Zephyr's :ref:`zephyr:logging_api`.
+  This option is enabled by default.
+* :option:`CONFIG_ZBOSS_TRACE_BINARY_LOGGING` - Stack logs are printed in the binary format using one of the following independent serial backends of your choice:
+
+  * :option:`CONFIG_ZBOSS_TRACE_UART_LOGGING` - UART serial.
+    This backend is enabled by default.
+  * :option:`CONFIG_ZBOSS_TRACE_USB_CDC_LOGGING` - USB CDC serial.
+
+  To specify the serial device, set :option:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME`.
+  By default, this option is set to the following values:
+
+  * ``UART_1`` if :option:`CONFIG_ZBOSS_TRACE_UART_LOGGING` is selected.
+  * ``CDC_ACM_1`` if :option:`CONFIG_ZBOSS_TRACE_USB_CDC_LOGGING` is selected.
+
+* :option:`CONFIG_ZBOSS_TRACE_BINARY_NCP_TRANSPORT_LOGGING` - Stack logs are printed in the binary format using the NCP transport channel.
+
+Stack logs are stored in the internal buffer if they are printed using Zephyr's :ref:`zephyr:logging_api` or the independent serial backend.
```

Comment:
When they are not buffered?
The NCP transport? What about the `static uint8_t uart_tx_buf_mem[CONFIG_ZIGBEE_UART_TX_BUF_LEN];` ?

Reply:
Yes, they are buffered, just don't use the same buffer. Will update this